src/omics/ugbio_omics/retrieve_from_glacier.py

```diff
@@ -0,0 +1,57 @@
+#!/bin/env python
+# This script is used to parse a WDL and JSON file and check if any files are in GLACIER.
+# If any files are in GLACIER, it will start the retrieval process.
+import argparse
+
+import boto3
+import winval.cloud_files_validator as cfv
+
+
+# parse input arguments
+def parse_arguments():
+    parser = argparse.ArgumentParser(
+        description="Parse JSON and WDL, check if any file in glacier, start retrieval if asked."
+    )
+    parser.add_argument("--wdl", type=str, required=True, help="The WDL to parse.")
+    parser.add_argument("--param_json", type=str, required=True, help="The parameter file.")
+    parser.add_argument("--retrieve", action="store_true", help="Retrieve the missing files from GLACIER.")
+    parser.add_argument("--n_days", type=int, default=90, help="GLACIER retrieval time in days. Default is 90 days.")
```

Comment:
I'd change the default to 30 days or even 7 days
I don't see a reason to restore files to so long time

Reply:
fixed